### ISSUE TITLE
Update to Tor v3 [$200]

### ISSUE THREAD

**Issue by woodser** [haveno-dex/haveno#123]
This issue requests updating Haveno to use Tor v3.

See relevant issues in Bisq for reference: https://github.com/bisq-network/projects/issues/23

**Comment by erciccione** [comment]
For this issue to be resolved, Tor v3 should be implemented across the platform, which should run without failures of any kind (related to the migration from Tor v2 to v3). Haveno on Tor v3 must run without issues on the 3 main operative systems (linux, mac, windows).

The kanban board used by Bisq to track their migration from v2 to v3 could be useful: https://github.com/orgs/bisq-network/projects/15

**Comment by l0nelyc0w** [comment]
Most of the migration is done automatically through the updated versions of the 3rd party dependencies (jtorctl etc) that have been updated to work with v3 onions.

I believe some minor modifications will be needed on haveno's side like checking for v3 onion string formatting instead of v2. I will check out behavior over the onion seednode I hosted just to be sure.

Bounty should probably be removed or decreased.
